### ISSUE TITLE
error: sorry something went wrong during download illustratorCC17.tgz

### ISSUE THREAD

**Issue by kupidr** [Gictorbit/illustratorCClinux#43]
![err](https://github.com/Gictorbit/illustratorCClinux/assets/96885329/8961ce90-1a5a-4e1b-9a81-04d160849bd5)

**Comment by ZaguePrime** [comment]
I've got the same issue

**Comment by ZaguePrime** [comment]
Alright the fix was in the earlier issues: on line 68 replace the victor link with :
https://downloads.runebase.io/illustratorCC17.tgz

**Comment by realquink** [comment]
Took me a second to find the file where to edit line 68, for anyone wondering it is here:

illustratorCClinux/scripts/illustratorSetup.sh

**Comment by IvanIsak2000** [comment]
> Alright the fix was in the earlier issues: on line 68 replace the victor link with : https://downloads.runebase.io/illustratorCC17.tgz

Yes, this solved the problem with the installation, but another error occurred
![image](https://github.com/Gictorbit/illustratorCClinux/assets/79650307/52f5b913-dcae-4847-8157-d180b13e5585)

**Comment by weiss-d** [comment]
@IvanIsak2000 first of all you shouldn't be doing regular stuff from under the Root user. Check out [this article](https://www.howtogeek.com/124950/htg-explains-why-you-shouldnt-log-into-your-linux-system-as-root/) if you're unsure why.

Next, try to repeat the install under a regular user and if the error persists, just manually create the missing folder it tries to put a desktop shortcut into.
Like this:
```shell
mkdir -p ~/.local/share/applications
```

**Comment by IvanIsak2000** [comment]
> @IvanIsak2000 first of all you shouldn't be doing regular stuff from under the Root user. Check out [this article](https://www.howtogeek.com/124950/htg-explains-why-you-shouldnt-log-into-your-linux-system-as-root/) if you're unsure why.
> 
> Next, try to repeat the install under a regular user and if the error persists, just manually create the missing folder it tries to put a desktop shortcut into. Like this:
> 
> ```shell
> mkdir -p ~/.local/share/applications
> ```

I think i solved this problem: i wrote bash script in root folder which start illustrator:
```
#!/bin/bash
sudo wine /root/.illustratorCC17/IllustratorCC17/IllustratorCC64.exe
```

**Comment by patrick-rayner** [comment]
> Alright the fix was in the earlier issues: on line 68 replace the victor link with : https://downloads.runebase.io/illustratorCC17.tgz

worked for me, thanks!

**Comment by TheredKnightt** [comment]
> > Alright the fix was in the earlier issues: on line 68 replace the victor link with : https://downloads.runebase.io/illustratorCC17.tgz
> 
> worked for me, thanks!

Sup, hope u can help me..just one question, how did u fix it? what I need to replace? I did not understand about the line 68

**Comment by TheredKnightt** [comment]
> Alright the fix was in the earlier issues: on line 68 replace the victor link with : https://downloads.runebase.io/illustratorCC17.tgz

Sup, hope u can help me..just one question, how did u fix it? what I need to replace? I did not understand about the line 68

**Comment by IvanIsak2000** [comment]
> > Alright the fix was in the earlier issues: on line 68 replace the victor link with : https://downloads.runebase.io/illustratorCC17.tgz
> 
> Sup, hope u can help me..just one question, how did u fix it? what I need to replace? I did not understand about the line 68

Hey, open to `scripts` folder and open `illustratorSetup.sh`, move to 68 line and replace value as `https://downloads.runebase.io/illustratorCC17.tgz`

**Comment by hamidmifdal** [comment]
Thank you all

**Comment by dad1755** [comment]
Really appreciate
